src/pages/home/live-pricing/_types.ts

```diff
@@ -0,0 +1,30 @@
+export type TMarketData = {
+    ask: number
+    bid: number
+    day_percentage_change: number
+    market: string
+    market_ranking: string
+    spread: number
+    symbol: string
+}
+
+export type TMarketDataResponse = {
+    trading_platform_asset_listing: any
+    active_symbols: TMarketData[]
+    req_id: number
+    error: any
```

Comment:
here as well, can we pass wrong data in api call and write here what we expecting?